patches/server/0010-Adventure.patch

```diff
@@ -4114,23 +4114,49 @@ index 3c3d5a527604d6f14abb1815816153747fcc0839..1b9a08098199f305606289724ee047e8
          return event;
      }
 diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java
-index 7cd70d53e5816938beab9019db40cd7faedba745..8ea031537c27b219c22316bff191670c48ebd602 100644
+index 7cd70d53e5816938beab9019db40cd7faedba745..53106cbe8034d759927469c4130573d681fcc6fc 100644
 --- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java
 +++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java
-@@ -72,6 +72,13 @@ public class CraftContainer extends AbstractContainerMenu {
+@@ -49,8 +49,8 @@ public class CraftContainer extends AbstractContainerMenu {
+     public CraftContainer(final Inventory inventory, final Player player, int id) {
+         this(new InventoryView() {
+ 
+-            private final String originalTitle = (inventory instanceof CraftInventoryCustom) ? ((CraftInventoryCustom.MinecraftInventory) ((CraftInventory) inventory).getInventory()).getTitle() : inventory.getType().getDefaultTitle();
+-            private String title = this.originalTitle;
++            private final net.kyori.adventure.text.Component originalTitle = inventory instanceof CraftInventoryCustom ? ((CraftInventoryCustom.MinecraftInventory) inventory).title() : inventory.getType().defaultTitle(); // Paper
++            private net.kyori.adventure.text.Component title = this.originalTitle;
+ 
+             @Override
+             public Inventory getTopInventory() {
+@@ -72,21 +72,23 @@ public class CraftContainer extends AbstractContainerMenu {
                  return inventory.getType();
              }
  
-+            // Paper start
-+            @Override
++            // Paper start - Adventure
+             @Override
+-            public String getTitle() {
+-                return title;
 +            public net.kyori.adventure.text.Component title() {
-+                return inventory instanceof CraftInventoryCustom ? ((CraftInventoryCustom.MinecraftInventory) ((CraftInventory) inventory).getInventory()).title() : net.kyori.adventure.text.Component.text(inventory.getType().getDefaultTitle());
-+            }
-+            // Paper end
-+
++                return this.title;
+             }
+ 
              @Override
-             public String getTitle() {
-                 return title;
+-            public String getOriginalTitle() {
+-                return originalTitle;
++            public net.kyori.adventure.text.Component originalTitle() {
++                return this.originalTitle;
+             }
+ 
+             @Override
+-            public void setTitle(String title) {
++            public void title(net.kyori.adventure.text.Component title) {
+                 CraftInventoryView.sendInventoryTitleChange(this, title);
+                 this.title = title;
+             }
++            //Paper end - Adventure
```

Comment:
Same situation here. Use the `// Paper` comments more strictly instead of wrapping lots of unchanged lines.